src/client/_clientwindow.py

```diff
@@ -870,12 +874,26 @@ def saveWindow(self):
         util.settings.setValue("geometry", self.saveGeometry())
         util.settings.endGroup()
 
+    def show_autojoin_settings_dialog(self):
+        autojoin_channels_list = []
+        autojoin_channels_list = Settings.get('chat/auto_join_channels')
+        text_of_autojoin_settings_dialog = """
+        Autojoin channels list : """ + ';'.join(autojoin_channels_list) + """
+        Enter the list of channels you want to join, separated by ;
+        For example: #poker;#newbie
+        To disable autojoin channels feature, leave the box empty and press ok.
```

Comment:
I'd do "To disable autojoining channels, leave the box empty and press OK."

Reply:
done